src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -162,6 +162,7 @@ public void ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled(bool
         [Fact]
         [SkipOnPlatform(TestPlatforms.OSX | TestPlatforms.MacCatalyst, "On OSX, ProcessName returns the script interpreter.")]
         [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS, "Not supported on iOS or tvOS.")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/73928", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/13757")]
```

Reply:
This is failing intermittently everywhere for last 3 years. Nothing to do with native AOT